Python/chapter01/1.8 - Zero Matrix/miguel_1.8_sol.py

```diff
@@ -0,0 +1,186 @@
+"""
+Python version 3.7.0
+1.8 - Zero Matrix
+Write an algorithm such that if an element in an MxN matrix is 0, its entire row and column are set to 0
```

Comment:
nit: wordwrap this comment and the one in L12.

Reply:
How exactly should I word-wrap the comments?

Reply:
Nvm I think I got it, lmk if the comments are better now

Reply:
i use vim, where the [`gqip`](https://vim.fandom.com/wiki/Automatic_word_wrapping) command does the trick. all other editors have similar ways of doing word-wrapping for you.

but yes, this looks much better!